modules/core/src-native/weaver/internals/Timestamp.scala

```diff
@@ -0,0 +1,35 @@
+package weaver.internals
+
+import scala.scalanative.posix
+import scala.scalanative.unsafe._
+
+import posix.time
+import posix.timeOps._
+
+private[weaver] object Timestamp {
+
+  def format(epochSecond: Long): String = Zone { implicit zone =>
+    val out     = alloc[time.tm]()
```

Comment:
I think this can be `stackalloc`, no? 

We're not return the pointer to the caller, so we don't need it on the heap, and the return value is a Scala String which will be on the heap anyways.

Reply:
And this makes Zone unnecessary as well

Reply:
yeah fair enough, I copied it verbatim from 0.4.0 code I had laying around

Reply:
Addressed [aa7296e](https://github.com/disneystreaming/weaver-test/pull/568/commits/aa7296e2be5a9ba2cd6ccad7cda27c5b76c76b81)